modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -176,26 +176,30 @@ class ApiClient(object):
             # use server/host defined in path or operation instead
             url = _host + resource_path
 
-        # perform request and return response
-        response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
-            method, url, query_params=query_params, headers=header_params,
-            post_params=post_params, body=body,
-            _preload_content=_preload_content,
-            _request_timeout=_request_timeout)
-
-        self.last_response = response_data
-
-        return_data = response_data
-        if _preload_content:
-            # deserialize response data
-            if response_type:
-                return_data = self.deserialize(
-                    response_data,
-                    response_type,
-                    _check_type
-                )
-            else:
-                return_data = None
+        try:
+            # perform request and return response
+            response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
+                method, url, query_params=query_params, headers=header_params,
+                post_params=post_params, body=body,
+                _preload_content=_preload_content,
+                _request_timeout=_request_timeout)
+
+            self.last_response = response_data
+
+            return_data = response_data
+            if _preload_content:
+                # deserialize response data
+                if response_type:
+                    return_data = self.deserialize(
+                        response_data,
+                        response_type,
+                        _check_type
+                    )
+                else:
+                    return_data = None
+        except ApiException as e:
+            e.body = e.body.decode('utf-8') if six.PY3 else e.body
```

Comment:
Style nit (not necessary to change):
```
if six.PY3:
  e.body = e.body.decode('utf-8')
```
For the else case we don't need to reassign the value of e.body